src/game/laramisc.c

```diff
@@ -508,74 +518,74 @@ void InitialiseLaraInventory(int32_t level_num)
 {
     Inv_RemoveAllItems();
 
-    START_INFO *start = &g_GameInfo.start[level_num];
+    RESUME_INFO *resume = Lara_GetResumeInfo(level_num);
```

Comment:
is the `resume` name clear enough? Wasn't sure since it could be start or current.